app/registries/foreman/settings/provisioning.rb

```diff
@@ -0,0 +1,200 @@
+      default: nil,
+      full_name: N_('Host owner'),
+      collection: proc { owner_select },
+      include_blank: N_("Select an owner"))
+    setting('root_pass',
+      type: :string,
+      description: N_("Default encrypted root password on provisioned hosts"),
+      default: nil,
+      full_name: N_('Root password'),
+      encrypted: true)
+    setting('unattended_url',
+      type: :string,
+      description: N_("URL hosts will retrieve templates from during build, when it starts with https unattended/userdata controllers cannot be accessed via HTTP"),
+      default: unattended_url,
+      full_name: N_('Unattended URL'))
+    setting('safemode_render',
+      type: :boolean,
+      description: N_("Enable safe mode config templates rendering (recommended)"),
+      default: true,
+      full_name: N_('Safemode rendering'))
+    setting('access_unattended_without_build',
+      type: :boolean,
+      description: N_("Allow access to unattended URLs without build mode being used"),
+      default: false,
+      full_name: N_('Access unattended without build'))
+    setting('manage_puppetca',
+      type: :boolean,
+      description: N_("Foreman will automate certificate signing upon provision of new host"),
+      default: true,
+      full_name: N_('Manage PuppetCA'))
+    setting('ignore_puppet_facts_for_provisioning',
+      type: :boolean,
+      description: N_("Stop updating IP address and MAC values from Puppet facts (affects all interfaces)"),
+      default: false,
+      full_name: N_('Ignore Puppet facts for provisioning'))
+    setting('ignored_interface_identifiers',
+      type: :array,
+      description: N_("Skip creating or updating host network interfaces objects with identifiers matching these values from incoming facts. You can use * wildcard to match identifiers with indexes e.g. macvtap*. The ignored interfaces raw facts will be still stored in the DB, see the 'Exclude pattern' setting for more details."),
+      default: IGNORED_INTERFACES,
+      full_name: N_('Ignore interfaces with matching identifier'))
+    setting('ignore_facts_for_operatingsystem',
+      type: :boolean,
+      description: N_("Stop updating Operating System from facts"),
+      default: false,
+      full_name: N_('Ignore facts for operating system'))
+    setting('ignore_facts_for_domain',
+      type: :boolean,
+      description: N_("Stop updating domain values from facts"),
+      default: false,
+      full_name: N_('Ignore facts for domain'))
+    setting('query_local_nameservers',
+      type: :boolean,
+      description: N_("Foreman will query the locally configured resolver instead of the SOA/NS authorities"),
+      default: false,
+      full_name: N_('Query local nameservers'))
+    setting('token_duration',
+      type: :integer,
+      description: N_("Time in minutes installation tokens should be valid for, 0 to disable token generation"),
+      default: 60 * 6,
+      full_name: N_('Token duration'))
```

Comment:
Also poorly named. Perhaps out of scope, but 
```suggestion
      full_name: N_('Installation token lifetime'))
```